feature_engine/transformation/yeojohnson.py

```diff
@@ -69,7 +69,11 @@ class YeoJohnsonTransformer(BaseNumericalTransformer):
 
     References
     ----------
-    .. [1] Weisberg S. "Yeo-Johnson Power Transformations".
+    .. [1] Yeo, In-Kwon and Johnson, Richard (2000).
+       A new family of power transformationsto improve normality or symmetry.
```

Comment:
almost there... could you please add a space between "transformations" and "to"?

Reply:
Oops, didn't notice the mistake. I've added proper spacing.
Thanks.